### ISSUE TITLE
Unable to load plugin

### ISSUE THREAD

**Issue by jrozner** [makigumo/MIPSCPU#1]
I'm able to successfully build the plugin and it installs itself into the correct directory in Application Support but when I start up Hopper the plugin isn't listed in the Plugins list.

**Comment by makigumo** [comment]
You should see some output like the following when run from the terminal:
```sh
$ /Applications/Hopper\ Disassembler\ v4.app/Contents/MacOS/Hopper\ Disassembler\ v4
2017-04-26 11:21:26.289 Hopper Disassembler v4[3983:157623] Hopper Disassembler v4.1.3 started
[..]
2017-04-26 11:21:26.301 Hopper Disassembler v4[3983:157623] Looking for CPU plugins into /Library/Application Support/Hopper/PlugIns/v4/CPUs
[..]
2017-04-26 11:21:26.304 Hopper Disassembler v4[3983:157623] Loading /Users/$USER/Library/Application Support/Hopper/PlugIns/v4/CPUs/MIPSCPU.hopperCPU
```

You probably also want to try the `reimpl` branch, where you can choose between two disassembler engines, one with limited instruction support (MIPS32) but better address handling, the other is based on capstone but the address handling is a little messed up :/

**Comment by jrozner** [comment]
Just tried this in the reimpl branch and I'm seeing this:
ERROR: Cannot load /Users/joe/Library/Application Support/Hopper/PlugIns/v4/CPUs/MIPSCPU.hopperCPU, plugin makes use of external function(s), and may be too slow for a CPU plugin. Also, make sure that plugin is compiled for OS X 10.9, or higher.

**Comment by makigumo** [comment]
This is interesting, when building with `xcodebuild` I'm getting the same error. When building from Xcode (8.3.2) it works fine. I'm not sure why though.

**Comment by jrozner** [comment]
I'm seeing similar behavior. I just tried building from within XCode itself. It looks like it's trying to load it but now I'm seeing this error:
```
2017-04-26 14:18:56.080 Hopper Disassembler v4[86224:2322157] Missing implementation of method "cpuContextClass" in "MIPSCPU" to conform protocol "CPUDefinition"
2017-04-26 14:18:56.080 Hopper Disassembler v4[86224:2322157]   ERROR: Class MIPSCPU does not conform to protocol CPUDefinition.
```

I did an upgrade this morning to hopper. Not sure if that is responsible for this error or not.

**Comment by makigumo** [comment]
Yes there was an SDK change. The `reimpl` branch is already updated ( https://github.com/makigumo/MIPSCPU/commit/14f77fe6703ccda69812f3c5e95a23bc1a5663ee).

**Comment by jrozner** [comment]
Cool, that worked. Thanks!

**Comment by makigumo** [comment]
That's good.

I just checked with the sample M68kCPU plugin from the SDK and it is showing the same problem.
I could narrow the problem down to the `Release` configuration, e.g. `xcodebuild -configuration Debug` is working.

**Comment by makigumo** [comment]
Load issues were fixed in Hopper 4.2.0.
